expressjs/src/routes/WifiConnect.ts

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
+import express from 'express'
+import dns = require('dns')
+
+const router = express.Router()
+
+// Set Axios defaults
+const wifiAxios = axios.create({ timeout: 15000 })
+wifiAxios.defaults.baseURL = 'http://py-wifi-connect:9090/'
```

Comment:
removing host networking and using internal service DNS resolved the issue for connecting to py-wifi-connect container, works fine now